ViewModel/Utilities/DefaultCurveData.cs

```diff
@@ -8,13 +8,14 @@ namespace HEC.FDA.ViewModel.Utilities
     public static class DefaultCurveData
```

Comment:
These values are not arbitrary. Richard sent me all these values and wants them to be the default values. In theory they will produce a nice compute. This will help us in testing because we can just create the default objects and run the compute and we know what values we should get.